src/mlpack/methods/ann/loss_functions/dice_loss.hpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file dice_loss.hpp
+ * @author N Rajiv Vaidyanathan
+ *
+ * Definition of the dice loss function.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LOSS_FUNCTIONS_DICE_LOSS_HPP
+#define MLPACK_METHODS_ANN_LOSS_FUNCTIONS_DICE_LOSS_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * The dice loss performance function measures the network's
```

Comment:
Do you think you could add/cite a reference here like: http://campar.in.tum.de/pub/milletari2016Vnet/milletari2016Vnet.pdf?